README.md

```diff
@@ -276,39 +276,37 @@ Make a new release tag, performing a few checks.
 
 Usage: `make-tag.py <tag>`.
 
-gitian-verify
--------------
+guix-verify
+-----------
 
-A script to verify gitian deterministic build signatures for a release in one
-glance. It will print a matrix of signer versus build package, and a list of
-missing keys.
+A script to verify guix deterministic build signatures for a release in one
+glance. It will print a matrix of signer versus build package ("noncodesigned"
+and "all"), and a list of missing keys.
 
-To be able to read gitian's YAML files and verify PGP signatures, it needs the
-`pyyaml` and `gpg` modules. This can be installed from pip, for example:
+To be able to verify PGP signatures, it needs the `gpg` modules. This can be
+installed from pip, for example:
 
-```bash
-pip3 install --user pyyaml gpg
+```sh
+pip3 install --user gpg
 ```
-(or install the distribution package, in Debian/Ubuntu this is `python3-yaml` and `python3-gpg`)
 
-The `gpg` module requires the [gpgme](https://www.gnupg.org/software/gpgme/index.html) library
-which is usually present on Linux, and can be installed with `brew install gpgme` on macOS.
+(or install the distribution package, in Debian/Ubuntu this is `python3-gpg`).
 
-Example usage: `./gitian-verify.py -r 0.21.0rc5 -d ../gitian.sigs -k ../bitcoin/contrib/builder-keys/keys.txt`
+Example usage: `./guix-verify.py -r 23.0rc2 -d ../guix.sigs -k ../bitcoin/contrib/builder-keys/keys.txt`
```

Comment:
Not really a blocker, but would you like to update this to use 24.0?

Reply:
Sure! Updated.

Reply:
Should be OK now. Sorry for my carelessness.